src/styles/components/_component-link.scss

```diff
@@ -0,0 +1,12 @@
+.component-link {
+  display: block;
+  outline: none;
+  width: 100%;
+  height: 100%;
+  background-color: var(--background-color);
+
+  &:focus .component-card {
+    outline: none;
+    border: 2px solid var(--accent-color);
+  }
```

Comment:
Whoops! Here's the code from your new focus states instead, with the little rotate: 

```
&:focus .component-card {
    position: relative;
    z-index: 1;
    outline: 2px solid var(--accent-color);
    outline-offset: 2px;
    transform: rotate(2deg);
  }
```